core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -2980,6 +3094,24 @@ private void setAttributeFromEntry(SetAttributeEntry entry)
     return mBlockMaster.getWorkerInfoList();
   }
 
+  private void checkAndAppendAuditLog(String command, String srcPath, String dstPath, AuditContext context) throws AccessControlException {
```

Comment:
- why do we `throws AccessControlException` here?
- nit: I would call this method `appendAuditContext` as we don't check any state here. Checking `!mAsyncAuditLogWriter.isEnabled()` is perf optimization.